### PR TITLE
Version 2.3.2

### PR DESCRIPTION
**Pull request by vegaro**
- Bump `purchases-ios` to 3.12.8
    [3.12.8 Changelog here](https://github.com/RevenueCat/purchases-ios/releases/tag/3.12.8)
    [3.12.7 Changelog here](https://github.com/RevenueCat/purchases-ios/releases/tag/3.12.7)
    [3.12.6 Changelog here](https://github.com/RevenueCat/purchases-ios/releases/tag/3.12.6)
    [3.12.5 Changelog here](https://github.com/RevenueCat/purchases-ios/releases/tag/3.12.5)
    [3.12.4 Changelog here](https://github.com/RevenueCat/purchases-ios/releases/tag/3.12.4)
 - Bump `purchases-android` to 4.3.3
    [4.3.3 Changelog here](https://github.com/RevenueCat/purchases-android/releases/tag/4.3.3)
    [4.3.2 Changelog here](https://github.com/RevenueCat/purchases-android/releases/tag/4.3.2)

### CODE REVIEW

**Comment by vegaro** [comment]
On the Android side of things, looks like it's related to the Java version. I will set up sdkman next week to see what's up.

File: .circleci/config.yml [hunk 1]
```diff
@@ -89,7 +108,7 @@ jobs:
   ios-integration-test:
     description: "Run iOS integration tests for Flutter"
     macos:
-      xcode: 11.3.1
+      xcode: 13.1.0
```
Comment (beylmk) [inline .circleci/config.yml:111]: maybe we could have a variable for the xcode version?

File: RELEASING.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 1. Update versions in `VERSIONS.md`
-1. Update to the latest purchases-hybrid-sdk version in `plugin.xml`
+1. Update to the latest purchases-hybrid-common version in `plugin.xml`
```
Comment (beylmk) [inline RELEASING.md:2]: 👍 good catch

**Review by beylmk — APPROVED** [review]
⚡

**Pull request merged**